pkg/asset/machines/libvirt/machines.go

```diff
@@ -61,7 +61,7 @@ func Machines(config *types.InstallConfig, pool *types.MachinePool, role, userDa
 	return machines, nil
 }
 
-func provider(platform *libvirt.Platform, name string) *libvirtprovider.LibvirtMachineProviderConfig {
+func provider(config *types.InstallConfig, platform *libvirt.Platform, name string) *libvirtprovider.LibvirtMachineProviderConfig {
```

Comment:
the only use is the clustername, WDYT asking for just that?

Reply:
> the only use is the clustername, WDYT asking for just that?

Done with 61d2e55 -> 18ca912.